app/src/main/java/org/gateshipone/odyssey/playbackservice/managers/OdysseyNotificationManager.java

```diff
@@ -143,6 +146,11 @@ public synchronized void updateNotification(TrackModel track, PlaybackService.PL
             notificationStyle.setShowActionsInCompactView(1, 2);
             notificationStyle.setMediaSession(mediaSessionToken);
             mNotificationBuilder.setStyle(notificationStyle);
+            mNotificationBuilder.setContentTitle(mContext.getResources().getString(R.string.notification_sensitive_content_replacement));
+
+            //Build the public notification
+            Notification notificationPublic = mNotificationBuilder.build();
```

Comment:
You should create the public part after you call setShowWhen. Currently your "private notification" has now a timestamp.

Reply:
I removed the Timestamp in commit: https://github.com/gateship-one/odyssey/commit/66075a411864e045f4669d25d196015fc8e42b20

This change was already merged into master.

I thought that a time is not really useful on a music player notification, as the notification will